### DISCUSSION TITLE
Playbook won't run

### DISCUSSION THREAD

**Discussion by Mike-Zelixon** [alivx/CIS-Ubuntu-20.04-Ansible#54] [Q&A]
Hi, I'm very new to Ansible and I installed this repo to harden a Ubuntu VM. 
However, I keep getting an error every time I try to run a playbook. 
For instance, when I run "ansible-playbook -i hosts main.yml" i get an error:

```
[WARNING]: Unable to parse /etc/ansible/CIS-Ubuntu-20.04-Ansible/tasks/hosts as an inventory source
[WARNING]: No inventory was parsed, only implicit localhost is available
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'
[DEPRECATION WARNING]: 'include' for playbook includes. You should use 'import_playbook' instead. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
ERROR! 'block' is not a valid attribute for a Play

The error appears to be in '/etc/ansible/CIS-Ubuntu-20.04-Ansible/tasks/section_1_Initial_Setup.yaml': line 6, column 3, but may be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

# server. If this filesystem type is not needed, disable it.
- name: 1.1.1.1 Ensure mounting of cramfs filesystems is disabled
  ^ here
```

Can you please tell me what I'm doing wrong? I will highly appreciate it.

**Comment by alivx** [comment]
> Hi, I'm very new to Ansible and I installed this repo to harden a Ubuntu VM.
> However, I keep getting an error every time I try to run a playbook.
> For instance, when I run "ansible-playbook -i hosts main.yml" i get an error:
> 
> ```
> [WARNING]: Unable to parse /etc/ansible/CIS-Ubuntu-20.04-Ansible/tasks/hosts as an inventory source
> [WARNING]: No inventory was parsed, only implicit localhost is available
> [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'
> [DEPRECATION WARNING]: 'include' for playbook includes. You should use 'import_playbook' instead. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
> ERROR! 'block' is not a valid attribute for a Play
> 
> The error appears to be in '/etc/ansible/CIS-Ubuntu-20.04-Ansible/tasks/section_1_Initial_Setup.yaml': line 6, column 3, but may be elsewhere in the file depending on the exact syntax problem.
> 
> The offending line appears to be:
> 
> # server. If this filesystem type is not needed, disable it.
> - name: 1.1.1.1 Ensure mounting of cramfs filesystems is disabled
>   ^ here
> ```
> 
> Can you please tell me what I'm doing wrong? I will highly appreciate it.

You must select a inventory host (the host you want to apply the changes to it)
